src/validate.js

```diff
@@ -3100,89 +3103,171 @@ function F_VALIDATE(disabledHL) {
+				}
 			}
-		}
-
-		// check if the venue was already seen
-		if (seen) {
-			HLObject(rawVenue);
-			continue;
-		}
-
-		///////////////////////////////////////////////////////////////////
-		// Prepare simple objects
-		var venue = new SimpleOBJECT(venueID, WMo.venues);
-		Object.seal(venue);
-
-		// mark venue as seen
-		_RT.$seen[venueID] = seen = [0, null, false, false,
-			4 > currentZoom,
-			cityID];
-
-		// increase city counter
-		venue.incCityCounter();
-
-		// shortcuts
-		var address = venue.$address;
-		var country = address.$country;
-		var countryLen = country.length;
-		var countryCode = country ? _I18n.getCountryCode(country.toUpperCase())
-			: _RT.$cachedTopCCode;
-		var city = address.$city;
-		var cityLen = city.length;
-		var cityID = address.$cityID;
-		var street = address.$street;
-		var state = address.$state;
-		var streetLen = street.length;
-		var alts = venue.$alts;
 
-		var exceptedCategories ='BRIDGE|ISLAND|FOREST_GROVE|SEA_LAKE_POOL|RIVER_STREAM|CANAL|DAM|TUNNEL|JUNCTION_INTERCHANGE'.split('|');
-
-		if (!cityLen
-			&& exceptedCategories.indexOf(venue.$categories[0]) === -1
-			&& isLimitOk(250)
-			&& address.isOkFor(250))
-			venue.report(250);
+			// check if the venue was already seen
+			if (seen) {
+				HLObject(rawVenue);
+				continue;
+			}
 
-	} // for all venues
+			///////////////////////////////////////////////////////////////////
+			// Prepare simple objects
+			var venue = new SimpleOBJECT(venueID, WMo.venues);
+			Object.seal(venue);
+
+			// mark venue as seen
+			_RT.$seen[venueID] = seen = [0, null, false, false,
+				4 > currentZoom,
+				cityID];
+
+			// increase city counter
+			venue.incCityCounter();
+
+			// shortcuts
+			var address = venue.$address;
+			var country = address.$country;
+			var countryLen = country.length;
+			var countryCode = country ? _I18n.getCountryCode(country.toUpperCase())
+				: _RT.$cachedTopCCode;
+			var city = address.$city;
+			var cityLen = city.length;
+			var cityID = address.$cityID;
+			var street = address.$street;
+			var state = address.$state;
+			var streetLen = street.length;
+			var alts = venue.$alts;
+			var lock = venue.$lock;
+
+			var exceptedCategories ='BRIDGE|ISLAND|FOREST_GROVE|SEA_LAKE_POOL|RIVER_STREAM|CANAL|DAM|TUNNEL|JUNCTION_INTERCHANGE'.split('|');
+
+			if (!cityLen
+				&& exceptedCategories.indexOf(venue.$categories[0]) === -1
+				&& isLimitOk(250)
+				&& address.isOkFor(250))
+				venue.report(250);
+
+			// GROUP name.length
+			if (!venue.$name.length) {
+				if (venue.$categories[0] === "PARK" && !cityLen
+					&& isLimitOk(258)
+					&& address.isOkFor(258))
+					venue.report(258);
+			}
+			// GROUP name.length
+			// Check for last update by bots
+			var botNamesAndIDs = [
+				'^waze-maint', '^105774162$',
+				'^waze3rdparty$', '^361008095$',
+				'^WazeParking1$', '^338475699$',
+				'^admin$', '^-1$',
+				'^avsus$', '^107668852$'
+			];
+			var re = new RegExp(botNamesAndIDs.join('|'),'i');
```

Comment:
Same here. The Regex should be created during the initialization. Looks like it belongs to i18n.